main.go

```diff
@@ -20,22 +20,25 @@ package main
 
 import (
 	"flag"
-	"github.com/loveholidays/ripley/pkg"
 	"os"
 	"runtime"
 	"runtime/pprof"
+
+	ripley "github.com/loveholidays/ripley/pkg"
 )
 
 func main() {
 	paceStr := flag.String("pace", "10s@1", `[duration]@[rate], e.g. "1m@1 30s@1.5 1h@2"`)
 	silent := flag.Bool("silent", false, "Suppress output")
 	dryRun := flag.Bool("dry-run", false, "Consume input but do not send HTTP requests to targets")
 	timeout := flag.Int("timeout", 10, "HTTP client timeout in seconds")
+	strictMode := flag.Bool("strictMode", false, "Strict mode")
```

Comment:
@eugenepaniot I'd change `strictMode` to `strict` and `"Strict mode"` to `"Panic on bad input"`

Reply:
done